src/webserver/Static.jl

```diff
@@ -222,6 +222,22 @@ function http_router_for(session::ServerSession)
     end
     HTTP.@register(router, "GET", "/notebookfile", serve_notebookfile)
 
+    serve_statefile = with_authentication(; 
+        required=security.require_secret_for_access || 
+        security.require_secret_for_open_links
+    ) do request::HTTP.Request
+        try
+            notebook = notebook_from_uri(request)
+            response = HTTP.Response(200, MsgPack.pack(Pluto.notebook_to_js(notebook)))
```

Comment:
This should be `pack` (i.e. `Pluto.pack`) instead of `MsgPack.pack`. Our packing and unpacking is slightly different from vanilla msgpack because:
- we added some extensions and 
- we don't convert `Int64` to `Int32`, `16` or `8` if it is small enough

https://github.com/fonsp/Pluto.jl/blob/main/src/webserver/MsgPack.jl
and the JS side:
https://github.com/fonsp/Pluto.jl/blob/main/frontend/common/MsgPack.js

This is actually something that you should look into for the WYSWYR project! You might want to:
- make sure that the client libraries have matching extensions, or
- use the unextended 'vanilla' MsgPack when sending responses, so that clients can use vanilla MsgPack. I think the first option is better

EDIT: actually my suggestion makes no difference, since `Pluto.pack(args...) = MsgPack.pack(args...)`, and our extensions to MsgPack are global (unfortunately). But still it makes sense 'semantically' to change it to `Pluto.pack`.